src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -218,6 +236,7 @@ public class CoinstakeContext
         /// <summary>Instance logger.</summary>
         private readonly ILogger logger;
 
+        /// <summary>Mining loop.</summary>
         private IAsyncLoop mining;
```

Comment:
it is actually staking loop
let's rename it to `stakingLoop` and modify comment accordingly -> `Loop in which the node attempts to generate new POS blocks by staking coins from its wallet.`